vizro-core/src/vizro/static/css/button.css

```diff
@@ -8,41 +10,28 @@
   letter-spacing: var(--letter-spacing-body-link-02);
   line-height: var(--text-size-05);
   padding: var(--spacing-01) var(--spacing-03);
+  text-transform: none;
 }
 
-.button_primary:hover {
-  background: var(--fill-medium-emphasis);
-  text-decoration-line: underline;
-}
-
-.button_primary:focus {
-  box-shadow: 0 0 0 2px var(--focus-focus) inset;
+#page-container .button_primary:hover {
+  background: linear-gradient(
+      var(--state-overlays-contrast-hover),
+      var(--state-overlays-contrast-hover)
+    ),
```

Comment:
@nadijagraca - I can explain in our catch-up, but basically, we have been doing some of the hover states wrong 😄 It wasn't clear from the Figma specifications for me either though